recipe/meta.yaml

```diff
@@ -16,11 +16,15 @@ build:
   number: 1
   script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   script_env:
-    - MAX_JOBS=$CPU_COUNT
-    # Not compiling for 8.0;8.6;8.9;9.0+PTX to keep builds under 6 hours
-    - TORCH_CUDA_ARCH_LIST=8.0+PTX
+    # Limit MAX_JOBS in order to prevent runners from crashing
+    - MAX_JOBS=4
+    - TORCH_CUDA_ARCH_LIST=8.0;8.6;8.9;9.0+PTX
   skip: true  # [cuda_compiler_version in (undefined, "None")]
   skip: true  # [not linux]
+  skip: true  # [py==313]  # Skip until pytorch dependency on setuptools is fixed
```

Comment:
Remove after https://github.com/conda-forge/pytorch-cpu-feedstock/pull/276 is merged (remember to re-render manually afterwards).

```suggestion
```

Reply:
We have to rebuild to add those additional modules requested in #18. We can see if pytorch is ready for 3.13 then.